config/sync/pathauto.pattern.custom_path.yml

```diff
@@ -0,0 +1,22 @@
+uuid: e5841cb4-d066-4895-93c0-fe4023c24833
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: custom_path
+label: 'custom path'
```

Comment:
Shouldn't this have a more semantic name? Like `Recipe path`? Custom is a little too generic.

If you change, please don't just rename the label but also recreate the pattern with a better machine name (id).